src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.6.0
+
+Released 2023-Dec-13
```

Comment:
Mention/Highlight that this is the first stable release.

Reply:
I have pushed a change. It's a little unconventional but I think it's okay considering this is a special release. Let me know what you think.